src/test/scala/org/broadinstitute/dsde/firecloud/service/SubmissionServiceSpec.scala

```diff
@@ -20,18 +19,13 @@ class SubmissionServiceSpec extends ServiceSpec with SubmissionService {
     MockWorkspaceServer.stopWorkspaceServer()
   }
 
-  val localSubmissionsPath = FireCloudConfig.Rawls.submissionsPath.format(
+  val localSubmissionsPath = submissionsPath.format(
```

Comment:
Yes! It is critical that we remember we should not be using a Rawls endpoint as an orchestration endpoint.